File: nastyhacks.py
n=int(input())
for i in range(0,n):
  lst = list(input().split())
  r=int(lst[0])
  e=int(lst[1])
  c=int(lst[2])

  if(e-c==r):
    print("does not matter");
  
  elif(e-c>r):
    print("advertise");
  
  else:
    print("do not advertise") ;

File: nastyhacks.rb
n=gets.to_i; 
for i in 1..n
  r,e,c=gets.chomp.split.map(&:to_f)
  if(e-c==r)
    puts "does not matter";
  
  elsif(e-c>r)
    puts "advertise";
  
  else
    puts "do not advertise";
  end 
end
